spec/03-plugins/23-rate-limiting/05-integration_spec.lua

```diff
@@ -363,123 +363,151 @@ describe("Plugin: rate-limiting (integration)", function()
           end
         end)
       end)
-    end -- for each redis strategy
+    end
```

Comment:
I noticed that the nesting was incorrect and those two tests executed in the for loop unnecessarily. I've just changed the nesting one level up.
I recommend viewing this file with the option to hide whitespace changes:
![image](https://github.com/Kong/kong/assets/5149215/5056b89b-c1eb-4201-b58b-b37e5f41e94a)